README.md

```diff
@@ -2496,7 +2496,7 @@ IEEE Conf. Computer Vision and Pattern Recognition Workshops (CVPRW), 2021. [Pro
 ### Human Pose Estimation
 - [Calabrese et al., CVPRW 2019](#Calabrese19cvprw),  
 *DHP19: Dynamic Vision Sensor 3D Human Pose Dataset*.
-- [Zhang et al., Neurocomputing 2023](#ZHANG2023126388),
+- [Zhang et al., Neurocomputing 2023](#Zhang23neurocomp),  
 *Neuromorphic high-frequency 3D dancing pose estimation in dynamic environment*, Neurocomputing, 547, 2023.
```

Comment:
No need to mention here "Neurocomputing, 547, 2023". It is already in the link "[Zhang et al. Neurocomputing 2023]"